addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4320,6 +4321,16 @@ registry.sizing = SnippetOptionWidget.extend({
      * @param {integer} [current] - current increment in this.grid
      */
     _onResize: function (compass, beginClass, current) {
+        this._updateSizingHandles(compass, beginClass, current);
+        this._notifyResizeChange();
+    },
+    /**
+     * @private
+     * @param {string} [compass] - resize direction ('n', 's', 'e' or 'w')
+     * @param {string} [beginClass] - attributes class at the beginning
+     * @param {integer} [current] - current increment in this.grid
+     */
+    _updateSizingHandles: function (compass, beginClass, current) {
```

Comment:
Where are those parameters used?

Reply:
Nowhere since last push force https://github.com/odoo/odoo/compare/6c9b10f08ec0724f90d6672a1e91492674fdfc8e..de668f626b5a56186d7fd69739280263c25a38c9#diff-ffb61e86e6b8297ef8997f9c605de14de896b54161c032d88a80bbaeac4f89beR4324
removed